src/templates/template-vision-classification/data.py

```diff
@@ -14,31 +14,27 @@ def setup_data(config: Any):
     """
     local_rank = idist.get_local_rank()
```

Comment:
Let's also put `it.use_dist` to local_rank definition. In non-dist config there is no need to define it